web/template/add_payment.gohtml

```diff
@@ -10,17 +10,17 @@
             {{ template "error-summary" .Error }}
 
             {{ if .Success }}
-                {{ template "success-banner" ("Payment added!") }}
+                {{ template "success-banner" ("Payment added") }}
             {{ end }}
 
-            <h1 class="govuk-heading-l app-!-embedded-hide">Add a payment</h1>
+            <h1 class="govuk-heading-l">Add a payment</h1>
 
             <form class="form" method="POST">
                 <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"/>
                 <div class="govuk-form-group">
                     <h3 class="govuk-label-wrapper">
```

Comment:
This `<h3>` tag isn't necessary and might confuse screenreaders (same on edit_payment.gohtml)